test/e2e/network/network_policy.go

```diff
@@ -54,7 +54,9 @@ type protocolPort struct {
 
 var protocolSCTP = v1.ProtocolSCTP
 
-var _ = SIGDescribe("NetworkPolicy [LinuxOnly]", func() {
+// Tagging these as serial because they should be run in isolation and differeint ginkgo.It() functions can
+// collide with each other due to the stateful nature of the polled pods in the suite.
+var _ = SIGDescribe("NetworkPolicy [LinuxOnly] [Serial]", func() {
```

Comment:
why?
The framework creates a new namespace per block so all of them should be isolated, tests should not share state
We should totally avoid Serial

Reply:
that's fair, I guess we can get rid of serial if you want, just spoke to @mattfenwick 
it may change some of the performance characterstics but overall, we still are moving forward , and will allow us to iterate later .

Reply:
Those tests can run in parallel, only Serial tests that can not run in parallel must have the tag.
People can use ginkgo.skip if they don't want to run them

Reply:
👍